contracts/contracts/coordination/subscription/EncryptorSlotsSubscription.sol

```diff
@@ -88,7 +88,11 @@ abstract contract EncryptorSlotsSubscription is AbstractSubscription {
             usedEncryptorSlots += addresses.length;
             require(usedEncryptorSlots <= encryptorSlots, "Encryptors slots filled up");
         } else {
-            usedEncryptorSlots -= addresses.length;
+            if (usedEncryptorSlots >= addresses.length) {
+                usedEncryptorSlots -= addresses.length;
+            } else {
+                usedEncryptorSlots = 0;
+            }
```

Comment:
shouldn't we use instead a `require(usedEncryptorSlots >= address.length, "...")`

Reply:
I'd not want to change to error because from subscription point of view - I do not see any issue to deauthorize too much. And it's allow list job to check it, but it's not very very strong opinion

Reply:
Fair enough 👍